### DISCUSSION TITLE
Notifications hard to read

### DISCUSSION THREAD

**Discussion by givihuda** [dbeaver/dbeaver#18950] [Q&A]
Whenever I get a notification, for the last several versions of DBeaver, its colors blend in with text, making it hard to read.

![image](https://user-images.githubusercontent.com/74785256/216688261-2d90c21e-6091-47da-802c-b4b921cc5659.png)

I'm not sure what to do about this.

I'm on macOS Monterey (latest version) and DBeaver 22.3.3.

**Comment by E1izabeth** [comment]
Do you use DBeaver's theme settings or some plugin like DarkestDark?

**Reply by givihuda** [reply]
I don't use a plugin. I haven't touched DBeaver's theme setting, I thought it just defaulted to whatever the Mac's was.

Now what?

**Reply by E1izabeth** [reply]
I suppose it's a bug and created an issue for you #18971
